packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -216,6 +230,140 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
 
   const userId = toFlatCommunicationIdentifier(adapter.getState().userId);
 
+  /* @conditional-compile-remove(rich-text-editor-image-upload) */
+  const fetchBlobData = useCallback(
```

Comment:
Should we ask Contoso to provide this functionality like it's done for the attachmentsfunctionality (`AttachmentOptions`)?